be/database/migrations/2023_07_21_025145_create_review_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('review', function (Blueprint $table) {
+            $table->id();
+            $table->text('body');
+            $table->integer('rating');
+            $table->foreignId('client_id')->constrained('client')->cascadeOnDelete();
+            $table->foreignId('room_id')->constrained('room')->cascadeOnDelete();
+            $table->timestamps();
+        });
```

Comment:
```suggestion
    {
        Schema::create('review', function (Blueprint $table) {
            $table->id();
            $table->foreignId('client_id')->constrained('client')->cascadeOnDelete();
            $table->foreignId('room_id')->constrained('room')->cascadeOnDelete();
            $table->text('body');
            $table->integer('rating');
            $table->timestamps();
        });
```
It is better to make the foreign keys above other columns so that when checking in database it is to track